opae.spec

```diff
@@ -79,15 +75,49 @@ This package contains OPAE tests.
 rm -rf mybuild
 mkdir mybuild
 cd mybuild
-%cmake .. -DBUILD_ASE=OFF
+%cmake .. -DBUILD_ASE=OFF -DOPAE_INSTALL_RPATH=OFF
 make -j
 
 %install
+mkdir -p %{buildroot}%{_datadir}/opae
+cp RELEASE_NOTES.md %{buildroot}%{_datadir}/opae/RELEASE_NOTES.md
+
+mkdir -p %{buildroot}%{_usr}/src/opae/cmake/modules
+for s in FindDBus.cmake \
+         FindGLIB.cmake \
+         FindOPAE.cmake \
+         FindQuartus.cmake \
+         FindQuesta.cmake \
+         FindRT.cmake \
+         FindUUID.cmake \
+         FindVerilator.cmake \
+         Findjson-c.cmake \
+         cmake_useful.cmake \
+         compiler_config.cmake \
+         libraries_config.cmake
+do
+  cp "cmake/modules/${s}" %{buildroot}%{_usr}/src/opae/cmake/modules
+done
+
+mkdir -p %{buildroot}%{_usr}/src/opae/samples
+cp samples/hello_fpga.c %{buildroot}%{_usr}/src/opae/samples/
+cp samples/hello_events.c %{buildroot}%{_usr}/src/opae/samples/
+cp samples/object_api.c %{buildroot}%{_usr}/src/opae/samples/
+
 cd mybuild
 make DESTDIR=%{buildroot} install
 mkdir -p %{buildroot}%{_sysconfdir}/systemd/system/
 mv %{buildroot}%{_usr}/lib/systemd/system/fpgad.service %{buildroot}%{_sysconfdir}/systemd/system/fpgad.service
 
+mkdir -p %{buildroot}%{_libdir}/opae/
+for l in libxfpga.so \
+         libmodbmc.so \
+         libfpgad-xfpga.so \
+         libfpgad-vc.so
```

Comment:
Can we add the opae prefix in the cmake install directive?
```cmake
install(TARGETS xfpga
        LIBRARY
        DESTINATION lib/opae
       )
```

Reply:
This worked out much better.  Thanks!